src/ports/postgres/modules/knn/test/knn.sql_in

```diff
@@ -0,0 +1,41 @@
+m4_include(`SQLCommon.m4')
+/* -----------------------------------------------------------------------------
+ * Test knn.
+ *
+ * FIXME: Verify results
```

Comment:
You mean to say that I should include assert statements in this test/knn.sql_in file in order to validate results, right?